github/files/drupal7/scripts/devops/reinstall/post_settings.yml

```diff
@@ -23,6 +23,18 @@
   when: item.status == true
   with_items: env.post_settings
 
+- name: Checking a domain_access is enabled
+  shell: 'drush pml --status=enabled -l {{ site_url }} | grep -c "(domain)" || true'
```

Comment:
@podarok I thought about using the `drush pml` command, but I'd  one issues:
The Domain module can be enabled two ways:
1. info file of profile, features, dependencies etc. (I had no problem with this one, we can use `drush pml` command)
2. Using a `*_env.yml` file (a `module` variable)
There is a bit problem, since the modules will be turned on after configuring the `settings.php` file. `pml` command will not work correctly.

I suggest that there is not entirely correct, enable `domain` module in `yml` file, but it is technically possible
is this second issue a critical?


P.S. Also we have a great risk of getting error messages if `domain_access` was enabled without `settings.php` fixes
```
WD domain: Domain access failed to load during phase: bootstrap include. Please check your settings.php file and site configuration.
Domain module installation is incomplete. See INSTALL.txt and check your settings.php file.
```
Maybe this code should be moved to `pre_settings.yml` or `pre_domain.yml` ...